golang/internal/grpc/grpc.go

```diff
@@ -129,15 +127,17 @@ func Init(grpcContext context.Context,
 
 	if grpcConn.Conn == nil {
 		var creds credentials.TransportCredentials
-		if connParams.insecure {
-			creds = insecure.NewCredentials()
-		} else {
-			httpAddr := fmt.Sprintf("https://%s", connParams.address)
-			certPool, err := fetchCertificatesFromURL(ctx, httpAddr)
-			if err != nil {
-				log.Error().Stack().Err(err).Msg("")
-			}
 
+		httpAddr := fmt.Sprintf("https://%s", connParams.address)
+		certPool, err := fetchCertificatesFromURL(ctx, httpAddr)
+		if err != nil {
+			if appConfig.Debug {
+				log.Warn().Err(err).Msg("Secure mode is disabled in demo/dev environment, falling back to plain-text gRPC")
```

Comment:
Error message starting with lower case in golang, nope? 🤔

Reply:
The error content is within `err` variable, that is the go error string, the message is a new construct, we could have some rules, but atm. we don't have any.

Reply:
The current ones okay for me.